model/database.py

```diff
@@ -0,0 +1,108 @@
+import torch
+import os
+from torch.utils.data import Dataset
+import numpy as np
+
+# Access environmental variables
+from dotenv import load_dotenv
+load_dotenv(verbose=True)
+
+# Get Pathname
+input_path = os.environ.get('pathname')
+output_path = os.environ.get('dataname')
+
+class DeNoiseDataset(Dataset):
+    """
+        Dataset that contains the real and generated spectrograms.
+    """
+
+    def __init__(self, data_folder):
+        """
+        Args:
+            data_folder (string): Path to the folder that contains everything.
+        """
+        self.generated = {}
```

Comment:
I think it may be better to store the file paths or some other unique identifier, then load the actual data in `__getitem__`.